chat-commands.js

```diff
@@ -3357,7 +3354,7 @@ exports.commands = {
 			this.sendReply('||<< ' + result);
 			/* eslint-enable no-unused-vars */
 		} catch (e) {
-			this.sendReply('|| << ' + ('' + e.stack).replace(/\n *at CommandContext\.eval [\s\S]*/m, '').replace(/\n/g, '\n||'));
+			this.sendReply(`|| << ${(`${e.stack}`).replace(/\n *at CommandContext\.eval [\s\S]*/m, '').replace(/\n/g, '\n||')}`);
```

Comment:
This looks like a syntax error

Reply:
It's not (eslint and therefore Travis can _definitely_ catch all syntax errors), but it's incredibly ugly. I'd probably break it out into its own line.